### ISSUE TITLE
Error when updating feature stack 

### ISSUE THREAD

**Issue by MEKH1** [fiji/Trainable_Segmentation#61]
Hello, 

I am trying to run the TWS automated in some image samples, but I got this error, how could I fix it?
any Idea will be appreciated.

Regards

![1](https://user-images.githubusercontent.com/43467769/100997759-fba2a080-355a-11eb-9aea-f88d06b5ff04.JPG)
![2](https://user-images.githubusercontent.com/43467769/100997875-1e34b980-355b-11eb-925b-1daa2c850737.JPG)

**Comment by iarganda** [comment]
@MEKH1 can you send me the full console error list? It seems the features didn't get calculated so maybe you hit your maximum reserved memory?

**Comment by MEKH1** [comment]
yes, sure Attached you can find it, I tried to increase the stack number, it worked on some images then, the error appeared again.  

[consol.txt](https://github.com/fiji/Trainable_Segmentation/files/5641766/consol.txt)
